src/validation_response.rs

```diff
@@ -5,7 +5,7 @@ use serde::{Deserialize, Serialize};
 #[derive(Serialize, Deserialize, Debug, Default)]
 #[serde(rename_all = "camelCase")]
 pub struct ValidationResponse {
-    pub uid: String,
+    pub uid: Option<String>,
```

Comment:
Why is this now optional, is that because Rego policies might not provide it back?

Reply:
Yes. This is the case, to unmarshal into `ValidationResponse` directly what's coming from the policy. We could split types and perform conversion between them, but I think is good enough. Many other things here are Optional already.

Reply:
I'm convinced about making `uid` optional. This is something that must always be set